.vscode/settings.json

```
@@ -13,7 +13,7 @@
     "editor.defaultFormatter": "esbenp.prettier-vscode"
   },
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true
+    "source.fixAll.eslint": "explicit"
```

Reverting this change as it's not related to the content at hand:

```suggestion
    "source.fixAll.eslint": true
```

If this is something that should be addressed, please open a separate change and add more context